integration-tests/src/test/resources/application.conf

```diff
@@ -13,6 +13,7 @@ datastax-java-driver {
     graph.timeout = 10 seconds
   }
   advanced {
+    session-leak.threshold = 16
```

Comment:
I'm not sure if this is the right value for integration test, but I'll run the suite in CI and we can adjust accordingly.